CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.10.3] - 2021-03-03
+
```

Comment:
I think the ideal solution would be to reset the build, maybe deleting the file directly on s3, but I'm not sure. But another solution could be just being specific on the changelog, in my opinion.
```suggestion

- Re-run project build
```